flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -47,10 +47,15 @@
 
 /**
  * A Transformer which combines a given list of input columns into a vector column. Types of input
- * columns must be either vector or numerical value.
+ * columns must be either vector or numerical types. The elements assembled in the same column must
+ * have the same size. If the element is null or has the wrong size, we will process this case with
```

Comment:
Let's avoid writing the JavaDoc using the first-person perspective. For example, instead of saying "we will process", let's say "the operator deals with null values or records with wrong sizes according to the strategy specified by the {@link HasHandleInvalid} parameter as follows".